Tests/test_SffIO.py

```diff
@@ -367,91 +367,93 @@ def test_index2(self):
-#         sff_trim = list(SffIterator(handle, trim=True))
-#
-#     from Bio import SeqIO
-#     filename = "Roche/E3MFGYR02_random_10_reads_no_trim.fasta"
-#     fasta_no_trim = list(SeqIO.parse(filename, "fasta"))
-#     filename = "Roche/E3MFGYR02_random_10_reads_no_trim.qual"
-#     qual_no_trim = list(SeqIO.parse(filename, "qual"))
-#
-#     filename = "Roche/E3MFGYR02_random_10_reads.fasta"
-#     fasta_trim = list(SeqIO.parse(filename, "fasta"))
-#     filename = "Roche/E3MFGYR02_random_10_reads.qual"
-#     qual_trim = list(SeqIO.parse(filename, "qual"))
-#
-#     for s, sT, f, q, fT, qT in zip(sff, sff_trim, fasta_no_trim,
-#                                    qual_no_trim, fasta_trim, qual_trim):
-#         # print("")
-#         print(s.id)
-#         # print(s.seq)
-#         # print(s.letter_annotations["phred_quality"])
-#
-#         assert s.id == f.id == q.id
-#         assert str(s.seq) == str(f.seq)
-#         assert s.letter_annotations[
-#             "phred_quality"] == q.letter_annotations["phred_quality"]
-#
-#         assert s.id == sT.id == fT.id == qT.id
-#         assert str(sT.seq) == str(fT.seq)
-#         assert sT.letter_annotations[
-#             "phred_quality"] == qT.letter_annotations["phred_quality"]
-#
-#     print("Writing with a list of SeqRecords...")
-#     handle = BytesIO()
-#     w = SffWriter(handle, xml=metadata)
-#     w.write_file(sff)  # list
-#     data = handle.getvalue()
-#     print("And again with an iterator...")
-#     handle = BytesIO()
-#     w = SffWriter(handle, xml=metadata)
-#     w.write_file(iter(sff))
-#     assert data == handle.getvalue()
-#     # Check 100% identical to the original:
-#     filename = "Roche/E3MFGYR02_random_10_reads.sff"
-#     with open(filename, "rb") as handle:
-#         original = handle.read()
-#         assert len(data) == len(original)
-#         assert data == original
-#         del data
-#
-#     print("-" * 50)
-#     filename = "Roche/greek.sff"
-#     with open(filename, "rb") as handle:
-#         for record in SffIterator(handle):
-#             print(record.id)
-#     with open(filename, "rb") as handle:
-#         index1 = sorted(_sff_read_roche_index(handle))
-#     with open(filename, "rb") as handle:
-#         index2 = sorted(_sff_do_slow_index(handle))
-#     assert index1 == index2
-#     try:
-#         with open(filename, "rb") as handle:
-#             print(ReadRocheXmlManifest(handle))
-#         assert False, "Should fail!"
-#     except ValueError:
-#         pass
-#
-#     with open(filename, "rb") as handle:
-#         for record in SffIterator(handle):
-#             pass
-#         try:
-#             for record in SffIterator(handle):
-#                 print(record.id)
-#             assert False, "Should have failed"
-#         except ValueError as err:
-#             print("Checking what happens on re-reading a handle:")
-#             print(err)
+
+class TestSelf(unittest.TestCase):
+    """
```

Comment:
Use a single line as per PEP257,

```
"""These tests were originally selftests run in SffIO.py"""
```

Also there was a spelling error.

If you think if a more meaningful class name, that'd be even better.